doc/usecases/bindings/UC_existing_application.md

```diff
@@ -0,0 +1,17 @@
+# Use Case: Programming Language Bindings in existing Application
+
+## Summary
+
+- **Scope:** API
+- **Level:** Underwater
+- **Actors:** API User
+- **Brief:** Developer wants to use libelektra in an existing application written in another language than C
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:** A binding for the programming language must exist and the maintainers of the application decided to use libelektra and the developer must have write access to the application.
+- **Main success scenario:** The application can be fully configured through libelektra
```

Comment:
A bit short, a more detailed scenario which describes multiple steps executed by the actor(s) and the expected outcomes would be a good improvement.